doc/LandIce/cori/do-cmake-albany-cori-gcc-knl-serial-sfad12

```diff
@@ -16,7 +16,6 @@ cmake \
       -D CMAKE_VERBOSE_MAKEFILE:BOOL=OFF \
       -D ENABLE_DEMO_PDES:BOOL=OFF \
       -D ENABLE_LANDICE:BOOL=ON \
-      -D ENABLE_KOKKOS_UNDER_DEVELOPMENT:BOOL=ON \
```

Comment:
Do you guys think we should keep cori config files? It's been decommissioned a long time ago...

Reply:
We could remove cori but maybe as part of another PR that removes all the other junk in "doc".

Reply:
Agreed. There's a lot of cleaning up needed int the doc folder after we got rid of Epetra.